TYPO3.Flow/Classes/TYPO3/Flow/Mvc/Controller/AbstractController.php

```diff
@@ -196,8 +196,6 @@ protected function forward($actionName, $controllerName = null, $packageKey = nu
         }
         if ($packageKey !== null) {
             $nextRequest->setControllerPackageKey($packageKey);
-        }
```

Comment:
This basically solves the case I guess but opens another bug if you don't set $packageKey but $subpackagekey because that won't be picked up then